repair_perf_test.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import random
+import datetime
+
+from sdcm.tester import ClusterTester
+from sdcm.db_stats import PrometheusDBStats
+
+
+class RepairPerfTest(ClusterTester):
+    """
+    Test performance of Repair related operations.
+    """
+
+    def test_bootstrap(self):
+        """
+        Init a basic cluster with 3 db nodes, and add a new node
+        """
+        stress_cmd = self.params.get('prepare_write_cmd')
+        stress = self.run_stress_thread(stress_cmd=stress_cmd)
+        self.verify_stress_thread(cs_thread_pool=stress)
+
+        stress_cmd = self.params.get('stress_cmd')
+        stress = self.run_stress_thread(stress_cmd=stress_cmd)
+
+        self.log.info("Adding new node to cluster...")
+        start = datetime.datetime.now()
```

Comment:
Consider to use `time.perf_counter()` for the purpose of the time elapsed measures: https://docs.python.org/3/library/time.html#time.perf_counter